dojo/settings/settings.dist.py

```diff
@@ -1040,6 +1040,8 @@ def saml2_attrib_map_format(dict):
     'Azure Security Center Recommendations Scan': ['unique_id_from_tool'],
     'Solar Appscreener Scan': ['title', 'file_path', 'line', 'severity'],
     'pip-audit Scan': ['vuln_id_from_tool', 'component_name', 'component_version'],
+    'Edgescan Scan': ['title', 'severity', 'cve', 'cwe'],
+    'Edgescan API Scan': ['title', 'severity', 'cve', 'cwe'],
```

Comment:
Does severity need to be in here? Usually there will be no 2 findings with the same title, same cve, same cwe but different severity?

Reply:
Good point. It has been reduced to just 'title' and 'severity'.

Reply:
Why does severity need to be in there?

Reply:
It is not always certain that our Findings will have a cve and cwe so I went with severity instead. Do you recommend removing severity altogether?

Reply:
This has been changed to just use `unique_id_from_tool`